packages/frontend-plugin-api/report.api.md

```diff
@@ -922,6 +870,13 @@ export { ErrorApiErrorContext };
 
 export { errorApiRef };
 
+// @public (undocumented)
+export const ErrorDisplay: ((
+  props: CoreErrorDisplayProps,
```

Comment:
Why is there a `Core` on props in here, when the components themselves don't have that in their names?